packages/cli-kit/src/index.ts

```diff
@@ -31,3 +31,4 @@ export * as vscode from './vscode.js'
 export * as outputMocker from './testing/output.js'
 export * as metadata from './metadata.js'
 export * as monorail from './monorail.js'
+export * as json from './json.js'
```

Comment:
We are moving away from the pattern of exporting everything from the `index.ts` so I'd advice not to add more imports here.

Reply:
Ah yes this is a remnant of something that we removed anyway. I'll remove it